src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const initialState = {
+  books: [],
+};
+
+export default function booksReducer(state = initialState, action) {
+  switch (action) {
+    case ADD_BOOK: {
+      return {
+        ADD_BOOK: true,
+      };
+    }
+    case REMOVE_BOOK: {
+      return {
+        REMOVE_BOOK: true,
+      };
```

Comment:
This should return a new array, which will not contain the book you want to remove from the store (filter by the id key - i.e.: return state.filter(book => book.id !== id); The function should use es6 filter method.